.gitignore

```diff
@@ -1,7 +1,13 @@
 .vs/
 .DS_Store
 **/*/.idea/
-*.dll
-*.dll.meta
```

Comment:
Any particular reason this was removed from the global?

Reply:
They're still in there. I just moved them down a bit with the other binary file types.